### PR TITLE
Use Async-OAuthlib rather than requests_oautlib which is not asynchronous library.

### PR DESCRIPTION
**Pull request by thec0sm0s**
Your fork (Quart-Discord) is using `requests_oauthlib` which is **NOT ASYNCHRONOUS** since it uses requests. That's why I forked `requests_oauthlib` and made it use `aiohttp` rather than requests which is fully asynchronous.

Hence merging this will with now finally turn this fork (Quart-Discord) into actually asynchronous library. Moreover there were multiple issues which were fixed along with this PR.

Also mind transfering Quart-Discord projects to me? I wish to keep maintaining both Flask-Discord and Quart-Discord. Thanks!

### CODE REVIEW

**Comment by jnawk** [comment]
👌 that sounds like a stellar idea.  I'll look into it when I get to a PC 👍

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Quart-Discord
-[![PyPI](https://img.shields.io/pypi/v/Quart-Discord?style=for-the-badge)](https://pypi.org/project/Quart-Discord/) [![Read the Docs](https://img.shields.io/readthedocs/quart-discord?style=for-the-badge)](https://quart-discord.readthedocs.io/en/latest/) 
+[![PyPI](https://img.shields.io/pypi/v/Flask-Discord?style=for-the-badge)](https://pypi.org/project/Flask-Discord/) [![Read the Docs](https://img.shields.io/readthedocs/flask-discord?style=for-the-badge)](https://flask-discord.readthedocs.io/en/latest/) [![Discord](https://img.shields.io/discord/690878977920729177?label=Discord%20Community&logo=Discord&style=for-the-badge)](https://discord.gg/7CrQEyP)
```
Comment (jnawk) [inline README.md:2]: Was this change quart -> flask intentional?

Comment (thec0sm0s) [reply README.md:2]: Nope. You may ignore the changes made in markdowns, docs or setup.py.

Comment (jnawk) [reply README.md:2]: You'll have to help me out here - as far as I understand it, accepting this PR will make all the changes, including the ones you say to ignore...  Do I misunderstand?

**Review by thec0sm0s — COMMENTED** [review]
This should do the work

File: README.md (hunk 1 above)
Comment (thec0sm0s) [reply README.md]: Cherry picked from your HEAD

**Pull request merged**